### PR TITLE
CSS Lesson Page

### PR DESCRIPTION
**Pull request by AVERGNET**

### CODE REVIEW

**Review by angela139 — APPROVED** [review]
LGTM!

File: pages/hack/css.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+- **Display**: `display` - Changing how elements are displayed (block, inline, flex, grid, etc.) and how they interact with other elements.
+- **Positioning**: `position`, `top`, `bottom`, `left`, `right` - Controlling the positioning of elements on the page using properties like position, top, bottom, left, right, etc.
+- **Float**: `float` - Floating elements to either the left or right side of their container.
+- **Opacity**: `opacity` - Adjusting the transparency of elements and their content.
+- **Transitions and Animations**: `transition`, `animation` - Applying smooth transitions and animations to elements.
+- **Shadows**: `box-shadow`, `text-shadow` - Adding box shadows or text shadows to elements for visual effects.
+- **List Styles**: `list-style-type`, `list-style-image`, `list-style-position` - Customizing the appearance of lists (unordered and ordered) with various list-style-type properties.
+- **Flexbox and Grid**: `display: flex`, `display: grid`, etc. - Utilizing CSS layout systems like Flexbox and Grid to create responsive and flexible layouts.
+
```
Comment (khushijpatel) [inline pages/hack/css.mdx:133]: Maybe we can add example codes using the common properties so students can see its usage and run on their end by changing the values to explore. Same for adding examples for flexbox.

**Review by khushijpatel — COMMENTED** [review]
Love the descriptions for the different properties!

**Comment by charvishukla** [comment]
10/10

**Pull request closed without merging**